oak-core/src/main/java/org/apache/jackrabbit/oak/query/RuntimeNodeTraversalException.java

```diff
@@ -0,0 +1,8 @@
+package org.apache.jackrabbit.oak.query;
+
+public class RuntimeNodeTraversalException extends RuntimeException {
```

Comment:
Could you extend UnsupportedOperationException? That would make it compatible with the previously thrown exception, UnsupportedOperationException. (Some code is / was catching UnsupportedOperationException.)

Reply:
Done